### ISSUE TITLE
[BUG] - Enabling printing & notifications doesn't unhide their Settings pages

### ISSUE THREAD

**Issue by Flearz** [Atlas-OS/Atlas#1101]
### Description

Hello,
I think i checked "Disable printing" on playbook install (0.4).

Enable Printing.cmd doesn't fully enable printing back, the windows menus are not working.

Running windows 10 10.00.19045 Build 19045

Thanks in advance

### Steps to reproduce

1. Clicking on this:
![image](https://github.com/Atlas-OS/Atlas/assets/32389536/fd7da80c-cbcf-4e14-b08e-1e89743532db)
2. Just always brings me there:
![image](https://github.com/Atlas-OS/Atlas/assets/32389536/78bd0b8f-4a98-4c57-b14e-0f370dd0c91c)

I tried to run it multiple times as admin.

### Expected behavior

Expect to get this menu:
![image](https://github.com/Atlas-OS/Atlas/assets/32389536/e573b936-fd25-4de1-b000-c75e109a92e7)

### Actual behavior

I get this
 ![image](https://github.com/Atlas-OS/Atlas/assets/32389536/78bd0b8f-4a98-4c57-b14e-0f370dd0c91c)

### Atlas Edition

Atlas for Windows 10 22H2

### Desktop information

Virtual machine:
![image](https://github.com/Atlas-OS/Atlas/assets/32389536/0319a254-c814-47b3-b4b9-8a5c5c3592fe)

**Comment by mian196** [comment]
If i remember correctly the printing to work on VM you need to install the vmware tools also

**Comment by mian196** [comment]
Also check the printing service manually in services

**Comment by he3als** [comment]
This is a known issue with the scripts for notifications and printing, it'll be fixed next update

**Comment by Znimator** [comment]
any workaround for this? I need to print my course work asap

**Comment by shelltar** [comment]
> any workaround for this? I need to print my course work asap

Printing still works, you just can't access the menu. I can print just fine. The scanner doesn't work, though.

**Comment by he3als** [comment]
To fix printing:
```
call "%windir%\AtlasModules\Scripts\settingsPages.cmd" /unhide printers
```
To fix notifications:
```
call "%windir%\AtlasModules\Scripts\settingsPages.cmd" /unhide notifications
call "%windir%\AtlasModules\Scripts\settingsPages.cmd" /unhide privacy-notifications
```

**Comment by robertosw** [comment]
I've got the same problem. Just installed a few days ago and I am 100% sure that I configured everything so that printing would still be possible after the AtlasOS install. 
If I search for "printer" I will get this:
![image](https://github.com/user-attachments/assets/1a3a1f65-2ab5-400d-80bc-128208093c03)
But clicking on this will just open the settings page on its home page. Devices > Printers does not exist

**Comment by woodendoors7** [comment]
In my case, the workaround does not actually unhide printers either, nor does it do stuff with notification - I can now search for printers in the search menu or settings search, but I cannot actually open the printers menu.

**Comment by Schokobecher** [comment]
This didn't work for me. I even checked _SettingsPageVisibility_ in _HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\Explorer_ and it reads:

`hide:recovery;autoplay;usb;maps;maps-downloadmaps;findmydevice;privacy;notifications;privacy-notifications;privacy-feedback;privacy-activityhistory;search-permissions;privacy-location;privacy-general;sync;;cortana-windowssearch;mobile-devices;mobile-devices-addphone;workplace;family-group;deviceusage;home;account`

Clicking anything print settings related just brings me back to the Settings Overwview.
Print Spooler is running, I also have the Print context-menu entry after running _Enable Printing_

Running AtlasOS v0.4.0 on Windows 11 Pro 23H2 (Version 10.0.22631 Build 22631)

**Comment by rdekens** [comment]
same issue her - fix above didnt fix.

**Comment by he3als** [comment]
Fixed in [release v0.4.1](https://github.com/Atlas-OS/Atlas/releases/tag/0.4.1). See the release for more info on what v0.4 users can do.
